config/stamp/stamp.yml

```diff
@@ -13,16 +13,16 @@ XQUEUE_HTTP_PORT: '8040'
 CERTS_QUEUE_URL: "http://localhost:{{ XQUEUE_HTTP_PORT }}"
 
 # Database connections - need to confirm connection strings
-ORA_MYSQL_HOST: "10.0.0.16"
-XQUEUE_MYSQL_HOST: "10.0.0.16"
-EDXAPP_MYSQL_HOST: "10.0.0.16"
-EDXAPP_MEMCACHE: ["10.0.0.16:11211"]
+ORA_MYSQL_HOST: "%%MYSQL_MASTER_IP%%"
+XQUEUE_MYSQL_HOST: "%%MYSQL_MASTER_IP%%"
+EDXAPP_MYSQL_HOST: "%%MYSQL_MASTER_IP%%"
+EDXAPP_MEMCACHE: ["%%MYSQL_MASTER_IP%%:11211"]
 MONGO_BIND_IP: "0.0.0.0"
 FORUM_MONGO_HOSTS:
-  - "10.0.0.13"
-  - "10.0.0.11"
-  - "10.0.0.12"
-EDXAPP_MONGO_HOSTS: ["%%MONGO_REPLICASET_NAME%%/10.0.0.11,10.0.0.12,10.0.0.13"]
+  - "%%MONGO_SERVER_IP_PREFIX%%11"
```

Comment:
I don't believe so. @satyarapelly , was there a reason why it was 13, 11, 12 instead of ordered?